Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 4
Rules claiming more threads will be scaled down.
Job stats:
job                  count
-----------------  -------
run_python_script        1
total                    1

Select jobs to execute...

[Wed Aug  7 11:42:27 2024]
rule run_python_script:
    input: /home/administrator/phd/PhagePleats_out/foldseek_out/search_result.tsv, data/presence_absence_c_40_tm_30_4100.csv, data/naive_bayes_classifier_Family.pkl, /home/administrator/phd/PhagePleats_out
    output: script_completed.txt
    jobid: 0
    reason: Code has changed since last execution
    resources: tmpdir=/tmp

Terminating processes on user request, this might take some time.
[Wed Aug  7 11:42:32 2024]
Error in rule run_python_script:
    jobid: 0
    input: /home/administrator/phd/PhagePleats_out/foldseek_out/search_result.tsv, data/presence_absence_c_40_tm_30_4100.csv, data/naive_bayes_classifier_Family.pkl, /home/administrator/phd/PhagePleats_out
    output: script_completed.txt

RuleException:
CalledProcessError in file /home/administrator/phd/PhagePleats/Snakefile, line 89:
Command 'mkdir -p /home/administrator/phd/PhagePleats_out &&  /home/administrator/programs/miniforge3/envs/CaudoAuto/bin/python3.10 /home/administrator/phd/PhagePleats/.snakemake/scripts/tmpqy8k4zo5.phagepleats.py' returned non-zero exit status 1.
  File "/home/administrator/phd/PhagePleats/Snakefile", line 89, in __rule_run_python_script
  File "/home/administrator/programs/miniforge3/envs/CaudoAuto/lib/python3.10/concurrent/futures/thread.py", line 58, in run
Complete log: .snakemake/log/2024-08-07T114227.739649.snakemake.log
